src/engine/test/engineTestMain.cpp

```diff
@@ -6,7 +6,7 @@
 int main(int argc, char** argv)
 {
     logging::LoggingConfig logConfig;
-    logConfig.logLevel = logging::LogLevel::Off;
+    logConfig.logLevel = "off";
     logging::loggingInit(logConfig);
 
     ::testing::InitGoogleTest(&argc, argv);
```

Comment:
What is the use of this file? Wasn't the main test removed?

Reply:
Yes, I will remove it.